pkg/sql/pgwire/conn.go

```diff
@@ -490,21 +490,10 @@ func (c *conn) handleSimpleQuery(
 			return nil
 		}
 
-		var sqlStr string
-		if len(stmts) == 1 {
-			// TODO(radu): the string could potentially contain a trailing semicolon.
-			// It's not trivial to trim the semicolon as it can be surrounded by
-			// whitespace, newlines, and/or comments.
-			sqlStr = query
-		} else {
-			// TODO(radu): return this information from the parser instead of using
-			// String.
-			sqlStr = stmt.String()
```

Comment:
This used the FmtSimple context which hides passwords and maybe other stuff by default. It looks like this change remove that which means things that were previously scrubbed on log files are no longer scrubbed.